### PR TITLE
Write UG for Plan Details and Plan Delete

### PR DESCRIPTION
**Pull request by TianaiYan**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -365,11 +366,92 @@ To view each plan in detail, enter
...
+To view each plan in detail, enter
+'plan /details <plan number in list>'.
+
+1. plan A
+2. plan B
+3. Grow My Muscles
+----------------------------------------------------------------------
+```
+To view details of plan number 3 from the list, enter the following command:
+*Plan number 3 is created by the command in [Create A Plan](#create-a-plan-plan-new).*
```
Comment (Musfirahe0556596) [inline docs/UserGuide.md:393]: nice linking :)!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -365,11 +366,92 @@ To view each plan in detail, enter
...
+```
+----------------------------------------------------------------------
+Here are the 4 workouts in [Grow My Muscles].
+1. push up (10 reps)
+2. sit up (10 reps)
+3. pull up (10 reps)
+4. push up (10 reps)
+----------------------------------------------------------------------
+```
+---
```
Comment (Musfirahe0556596) [inline docs/UserGuide.md:407]: Perhaps could add a short explanation on the expected outcome. E.g. "All workouts in the Grow My Muscles plan will be displayed....."

**Review by Musfirahe0556596 — APPROVED** [review]
Looks great! Some suggestions to consider ^_^

File: docs/UserGuide.md [hunk 3]
```diff
@@ -293,7 +294,7 @@ Alright, the following workout has been updated:
 
 ----------------------------------------------------------------------
 ```
-Now the workout with index 1 (push up with 10 reps) is updated to push up with 15 reps.
+Now the workout number 1 (push up with 10 reps) is updated to push up with 15 reps.
```
Comment (alanlowzies) [inline docs/UserGuide.md:297]: Perhaps a comma can be added after "Now" and the word "the" can be removed. 

> "Now, ~the~ workout number 1..."

**Review by alanlowzies — COMMENTED** [review]
Looks alright. Just one grammar-related suggestion for your consideration. Thank you

**Pull request merged**